src/controllers/CategoryController.js

```diff
@@ -55,8 +58,9 @@ export const updateCategory = catchAsync(async (req, res, next) => {
   });
```

Comment:
Is 'product key' a mistake here? Shouldn't it be 'category'? However, please do not return the key name, as this will make my work on the front-end more straightforward. Only return the result. Could you apply this change in all similar instances?